index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <link href="main.css" rel="stylesheet" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>This should be a title</title>
+    <script src="index.js"></script>
+
+  </head>
+  
+  <body>
+    <p class="hello-snake">Hello world</p>
+    <p>Not stylized</p> 
+    <button class="buttonStyle" onclick="helloLog()">Click me!</button>
```

Comment:
Could you do me a favor, and instead of calling through the HTMl, could you target the button, and have it work through the JS?